### DISCUSSION TITLE
Unable to work with useAnimations

### DISCUSSION THREAD

**Discussion by igneous8k** [pmndrs/drei#974] [Q&A]
Hi, i have a model file in glb format and use gltfjsx to convert the same into jsx via https://gltf.pmnd.rs/     .
If I comment the line which contains useAnimations in model file , model works fine but just by calling hook app crashes saying cannot redefine property ...
Please help me out in loading animations into the code im providing via file link below.
I appreciate your efforts. 

Model file gltf :
https://drive.google.com/file/d/1Qd8SqQ9cWwZwScithcAbMhyH1EniZKgX/view?usp=sharing

Code with glb file: 
https://drive.google.com/file/d/1WS93X12qf4ngkIabAcyK7g3N9cdPHYaC/view?usp=sharing

**Comment by drcmda** [comment]
you can check out fiber docs, there are examples that useAnimations. it's not doing much, creates a mixer, clips, and actions become available in useEffect, not at render time.
